app/src/componentLibrary/attributeList/editableAttribute/attribute/attribute.scss

```diff
@@ -42,6 +44,10 @@
   max-width: 100%;
   column-gap: 8px;
 
+  &:hover:not(:focus-visible) {
```

Comment:
Does this hover state need to appear in the disabled state?

Reply:
if there is focus but the hover should not work